morphman/common/common.py

```diff
@@ -127,17 +104,11 @@ def write_parameters(data, folder):
     parameters = get_parameters(folder)
 
     # Add new parameters (can overwrite old as well)
-    for key, value in list(data.items()):
-        parameters[key] = value
+    parameters.update(data)
 
-    # Get new text
-    text = ["%s: %s" % (k, v) for k, v in list(parameters.items())]
-    text = "\n".join(text)
-
-    # Write text
-    f = open(folder + "_info.txt", "w")
-    f.write(text)
-    f.close()
+    # Write data
+    with open(folder + "_info.txt", "w") as outfile:
```

Comment:
Burde kanskje lagres som `.json`-format?  Er vel ikke krise om det ikke er det, men da er det ingen tvil om formatet.

Reply:
Helt enig, jeg pushet for å fikset dette.